### DISCUSSION TITLE
Official install.sh script does update as well?

### DISCUSSION THREAD

**Discussion by MxD-js** [coder/code-server#3414] [Q&A] [answered]
Wanted to ask if I used this script for install, and on a new release of code-server will this script update an existing installation? I wanted to automate the updating process, either setting up a chron job to run the script, or something similar. 

`curl -fsSL https://code-server.dev/install.sh | sh
`

**Comment by code-asher** [comment, marked as answer]
Yup, it'll update an existing installation to the latest version available.

Note that code-server will have to be restarted after installing the update.
